homeassistant/components/homee/sensor.py

```diff
@@ -84,11 +91,6 @@ class HomeeSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.POWER,
         state_class=SensorStateClass.MEASUREMENT,
     ),
-    AttributeType.CURRENT_VALVE_POSITION: HomeeSensorEntityDescription(
```

Comment:
Please don't mix different things in the same PR. Removing a feature shouldn't be part of a fix or a feature addition.